lightstep/resource_metric_condition.go

```diff
@@ -574,6 +583,18 @@ func buildQueries(queriesIn []interface{}, includeSpansQuery bool) ([]client.Met
 			},
 		}
 
+		timeseriesOperatorInputWindowMs := query["timeseries_operator_input_window_ms"]
+		if timeseriesOperatorInputWindowMs != 0 { // 0 here indicates that the value was not set in terraform file
+			value := timeseriesOperatorInputWindowMs.(int)
+			newQuery.Query.TimeseriesOperatorInputWindowMs = &value
+		}
+
+		if newQuery.Type == "single" {
```

Comment:
Do we also need to check whether it's a metrics or spans alert to know whether to set `newQuery.Query` vs `newQuery.SpansQuery`? I think you mentioned that you were just opportunistically adding some support for spand queries, so feel free to just add a TODO if this comment isn't part of the current scope of work (which I think it's not)

Reply:
Correct. This PR is not attempting to support span alerts.